go/libkb/upgrade_instructions.go

```diff
@@ -36,15 +36,15 @@ func linuxUpgradeInstructions() {
 	}
 
 	if hasPackageManager("apt-get") {
-		printUpgradeCommand("sudo apt-get update && sudo apt-get install " + packageName)
+		printUpgradeCommand(g, "sudo apt-get update && sudo apt-get install "+packageName)
```

Comment:
I'm kind of surprised `gofmt` doesn't have an opinion about space around `+`. Is there any particular style guide we're following on this one?